runner/job_manager.py

```diff
@@ -90,6 +91,7 @@ def create_and_submit_job(self, batch_client: batch.BatchExtensionsClient):
         template = ctm.load_file(self.template_file)
         parameters = ctm.load_file(self.parameters_file)
 
+        utils.update_params_with_values_from_keyvault(parameters, self.keyvault_client_with_url)
         # overrides some of the parameters needed in the file, container SAS
```

Comment:
Can you add a small # comment above this?

Reply:
Believe its pretty self explanatory but will just for you